demisto_sdk/__main__.py

```diff
@@ -123,7 +123,7 @@ def get_context_arg(args):
         for arg in args:
             if type(arg) == click.core.Context:
                 return arg
-        print(
+        print(  # noqa: T201
```

Comment:
why not replace with logger here?

Reply:
It's a chicken and egg here.
I'm setting up the logger, so can't really use it here.